app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comment
+  has_many :like
```

Comment:
- [ ] As requested by the project requirements, the Post model should have a method that updates the posts counter for a user
- [ ] Also the Post model should have a method that returns the 5 most recent comments for a given post. Kindly add this methods